tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -158,23 +158,18 @@ by ``0.5`` to keep the line centered as shown below.
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    extends Node2D
-
     func _draw():
         draw_line(Vector2(1.5, 1), Vector2(1.5, 4), Color.GREEN, 1.0)
         draw_line(Vector2(4, 1), Vector2(4, 4), Color.GREEN, 2.0)
         draw_line(Vector2(7.5, 1), Vector2(7.5, 4), Color.GREEN, 3.0)
 
  .. code-tab:: csharp
 
-    public class CustomNode2D : Node2D
+    public override void _Draw()
     {
-        public override void _Draw()
-        {
-            DrawLine(new Vector2(1.5, 1), new Vector2(1.5, 4), Colors.Green, 1.0)
-            DrawLine(new Vector2(4, 1), new Vector2(4, 4), Colors.Green, 2.0)
-            DrawLine(new Vector2(7.5, 1), new Vector2(7.5, 4), Colors.Green, 3.0)
-        }
+        DrawLine(new Vector2(1.5, 1), new Vector2(1.5, 4), Colors.Green, 1.0);
+        DrawLine(new Vector2(4, 1), new Vector2(4, 4), Colors.Green, 2.0);
+        DrawLine(new Vector2(7.5, 1), new Vector2(7.5, 4), Colors.Green, 3.0);
```

Comment:
Float literals need the `f` suffix, otherwise they are doubles.

```suggestion
        DrawLine(new Vector2(1.5f, 1), new Vector2(1.5f, 4), Colors.Green, 1.0f);
        DrawLine(new Vector2(4, 1), new Vector2(4, 4), Colors.Green, 2.0f);
        DrawLine(new Vector2(7.5f, 1), new Vector2(7.5f, 4), Colors.Green, 3.0f);
```